docs/roundabout.md

```diff
@@ -0,0 +1,25 @@
+# Roundabout
+
+> Roundabout allows the creation of short lived presigned URLs for content stored in carpark bucket (in R2). It can also get the HTTP request redirected to the given presigned URL.
+
+## HTTP API
+
+The given API is currently public.
+
+### `GET /{carCid}`
+
+Gets the content of a CAR file (by its CID) from a signed URL directly to the bucket where it is being stored. Behind the scenes, this request will return a `302 Redirect` to a created signed URL.
+
+```console
+curl -L -v https://roundabout.web3.storage/presigned-url/bagbaiera222226db4v4oli5fldqghzgbv5rqv3n4ykyfxk7shfr42bfnqwua --output bagbaiera222226db4v4oli5fldqghzgbv5rqv3n4ykyfxk7shfr42bfnqwua.car
+```
+
+### `GET /presigned-url/{carCid}`
```

Comment:
Do we need this endpoint, could we not just send a HEAD request like `HEAD /${carCid}` and extract the `Location` header?